src/main/kotlin/org/ozwillo/dcimporter/repository/ProcessingStatRepository.kt

```diff
@@ -0,0 +1,88 @@
+package org.ozwillo.dcimporter.repository
+
+import org.ozwillo.dcimporter.model.ProcessingStat
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository
+import org.springframework.stereotype.Repository
+import reactor.core.publisher.Flux
+import java.time.LocalDateTime
+
+@Repository
+interface ProcessingStatRepository: ReactiveMongoRepository<ProcessingStat, String>{
```

Comment:
We can do dynamic research, using criterias or queryDSL 
Helpfull link : https://www.baeldung.com/queries-in-spring-data-mongodb

Reply:
As the "search" function have no utility yet I removed it. But I'll keep this in mind, thank you.